CMakeLists.txt

```diff
@@ -114,10 +114,10 @@ configure_file(${CMAKE_SOURCE_DIR}/versions.target.cmake ${CMAKE_CURRENT_BINARY_
 file(WRITE ${CMAKE_BINARY_DIR}/version.txt ${FDB_VERSION})
 
 set(FDB_CURRENT_VERSION ${PROJECT_VERSION})
-set(FDB_FUTURE_VERSION "7.4.0")
-set(FDB_PREV_RELEASE_VERSION "7.1.25")
-set(FDB_PREV2_RELEASE_VERSION "7.0.0")
-set(FDB_PREV3_RELEASE_VERSION "6.3.25")
+set(FDB_FUTURE_VERSION "71.4.0")
+set(FDB_PREV_RELEASE_VERSION "71.2.7")
```

Comment:
This might break something if we set it to this now? We can always change this as we do future 71.2 patches?
Also this appears to be for major release versions, so iirc this should be
71.2.6
7.1.4
7.0.1

Reply:
Oh yes, you're right, I think it is major releases. So we probably want 71.2.0 instead of 71.2.6/71.2.7?

Reply:
Oh yes, you're right, I think it is major releases. Updated accordingly.

Reply:
Please keep the prev release versions unchanged, because the tests download their binaries from the releases published in github, and 71.2.x releases are not available there. For many of the tests it does not matter much what is the specific previous release it is testing. The tests where it matters are commented out.